demo/admin/src/pages/mainMenu/components/EditMainMenuItem.tsx

```diff
@@ -109,7 +106,7 @@ const EditMainMenuItem: React.FunctionComponent<EditMainMenuItemProps> = ({ item
                 input: { content: content ? RichTextBlock.state2Output(content) : null },
                 lastUpdatedAt: item.updatedAt,
             },
-            refetchQueries: [namedOperations.Query.MainMenuItem],
```

Comment:
Why doesn't named operations work anymore?

Reply:
because MainMenuItem Query is defined in another file